### PR TITLE
add test cases

### PR DESCRIPTION
**Pull request by Parm-S**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -14,6 +14,8 @@ enum Theme {
 }
 
 const Header: React.FC = () => {
+  const setThemeName = React.useContext(ThemeContext);
+
```
Comment (nik-1207) [inline src/components/Header/Header.tsx:18]: unnecessary change

**Review by nik-1207 — CHANGES_REQUESTED** [review]

File: src/components/Header/Header.tsx (hunk 1 above)
Comment (Parm-S) [reply src/components/Header/Header.tsx:18]: ok

**Review by AbhayMaheshwari123 — APPROVED** [review]

File: src/components/Header/__tests__/Header.spec.tsx [hunk 2]
```diff
@@ -1,33 +1,42 @@
 import React from "react";
 import { Header } from "../Header";
 import { fireEvent, render, screen } from "@testing-library/react";
-import { ThemeContext } from "../../ThemeSetter";
+import { ThemeContext } from "../../ThemeContext";
 
-describe("testing Header", () => {
-  it("should render Header", () => {
+describe("Header", () => {
+  it("Header Match snapshot test", () => {
```
Comment (AbhayMaheshwari123) [inline src/components/Header/__tests__/Header.spec.tsx]: word "test" shouldn't be there in explanation.

**Review by AbhayMaheshwari123 — COMMENTED** [review]
single change review it

File: src/components/Header/__tests__/Header.spec.tsx (hunk 2 above)
Comment (Parm-S) [reply src/components/Header/__tests__/Header.spec.tsx]: okay

**Review by AbhayMaheshwari123 — APPROVED** [review]

**Review by AbhayMaheshwari123 — APPROVED** [review]
